bfvmm/include/hve/arch/intel_x64/vic.h

```diff
@@ -64,10 +64,10 @@ namespace proc_ctl2 = vmcs_n::secondary_processor_based_vm_execution_controls;
 /// Helpers
 ///-----------------------------------------------------------------------------
 
-template<
+template <
     typename N,
-    typename = std::enable_if_t<std::is_integral<N>::value ||
-                                std::is_pointer<N>::value>>
+    typename = std::enable_if_t < std::is_integral<N>::value ||
```

Comment:
I dont like gaps here either

Reply:
Templates should use this syntax to avoid this:

```
template <
    typename T,
    typename B,
    typename = std::enable_if<std::is_integral<T>::value>,
    typename = std::enable_if<std::is_integral<B>::value>
    >

```

Reply:
Although the use of || might introduce the spaces (once you convert to the provided syntax). If that is still the case, we should file a bug as that should not happen